docs/safe_tx_gas.md

```diff
@@ -1,14 +1,14 @@
 ### Safe Transaction Gas Limit (safeTxGas)
 
-In this document will describe the behaviour of the internal gas limit (aka `safeTxGas`) for the [1.3.0](https://github.com/safe-global/safe-smart-account/releases/tag/v1.3.0-libs.0) version of the Safe Smart Account.
+This document will describe the behaviour of the internal gas limit (aka `safeTxGas`) for the [1.3.0](https://github.com/safe-global/safe-smart-account/releases/tag/v1.3.0-libs.0) version of the Safe Smart Account.
```

Comment:
I think the sentence was meant to start with "In this document we describe ..."  The current sentence sounds weird (talking in the future about the current document)

Reply:
Updated in #857